myriadeploy/deployment.cfg.local

```diff
@@ -16,3 +14,8 @@ rest_port = 8753
 [workers]
 1 = localhost:9001
 2 = localhost:9002
+
+[runtime]
+# Uncomment to set the maximum heap size of the worker processes
+#max_heap_size = -Xmx2g
```

Comment:
Thanks for both suggestions, since we don't have other JVM flags to apply right now, I changed it to be "max_heap_size_gb" which takes a number, and also add "min_heap_size_gb". If we have more in the future, adding a "JVM flags" key can be a solution.